### DISCUSSION TITLE
Auto Z-offset?

### DISCUSSION THREAD

**Discussion by SalsShadow** [1coderookie/Klipper4KobraGoNeo#18] [Q&A]
Is the z-offset automatically set after probing? I know last time I tried to run klipper on this machine no config I found has auto z-offset working yet. This is for the neo config.

Thanks!

**Comment by 1coderookie** [comment]
This machine doesn't have an auto z-offset function, already not with the stock firmware (that would be the Kobra **2** Neo then), you gotta set the z-offset manually as usual and save it then.
